Reply/app/src/main/java/com/example/reply/ui/MainActivity.kt

```diff
@@ -42,6 +43,7 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         enableEdgeToEdge()
         super.onCreate(savedInstanceState)
+        ViewCompat.setOnApplyWindowInsetsListener(window.decorView) { _, insets -> insets }
```

Comment:
This gets the inset and return it without doing anything. It seems to me that it doesn't do what the before/after screenshots do.
Is PR missing some commits?

Reply:
No, it isn't.
The MDC Catalog app does the same thing for edge-to-edge layout.
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/windowpreferences/WindowPreferencesManager.java#L80-L81
https://github.com/material-components/material-components-android/blob/master/catalog/java/io/material/catalog/windowpreferences/WindowPreferencesManager.java#L41-L46

Reply:
https://github.com/android/compose-samples/pull/1265/commits/a1e7c597eb85cec716f5e83575a18b283278f6e2:
![Screenshot_20240220_150514](https://github.com/android/compose-samples/assets/71050561/b74dd2d7-d834-4c36-b634-adaf09b00c0a)